src/goto-instrument/cover.h

```diff
@@ -42,41 +42,46 @@ enum class coverage_criteriont
   PATH, MCDC, ASSERTION, COVER };
 
 bool consider_goals(
-  const goto_programt &goto_program,
-  coverage_goalst &goals);
+  const goto_programt &,
```

Comment:
It adds no information that the parameter that's the only goto_programt is "the goto program", i.e., this is visual noise.